docs/examples/blocks/slider.md

```diff
@@ -9,6 +9,12 @@ The currently selected value is in the attribute `value`.
 Don't change this value manually, but use the function `set_close_to!(slider, value)`.
 This is necessary to ensure the value is actually present in the `range` attribute.
 
+If the slider value is used for a relatively slow task, it may be more effective to
+use the attribute `value_dragstop` instead of `value`.  The `value_dragstop` is only
+updated when the mouse is released to conclude the slider drag operation.  It is 
+is synchronized with `value` when calling `set_cloe_to!(slider, value)` or one a mouse
```

Comment:
set_cloe_to

Reply:
this section should go before the example I think, with an appropriate subheading